include/commands.hpp

```diff
@@ -44,13 +44,19 @@ _base_channel				parseChannel(const std::string &channel);
 std::vector<std::string>	parseList(const std::string &list);
 
 int	command_join(Message &message, Dependencies &deps);
+int	command_mode(Message &message, Dependencies &deps);
 int	command_nick(Message &message, Dependencies &deps);
 int	command_part(Message &message, Dependencies &deps);
 int	command_pass(Message &message, Dependencies &deps);
 int	command_ping(Message &message, Dependencies &deps);
 int	command_quit(Message &message, Dependencies &deps);
 int	command_user(Message &message, Dependencies &deps);
 
+void	command_mode_channel(Message &message, Dependencies &deps, const _base_channel &base_channel);
+void	flag_operator(bool add_flag, User &author, Channel &channel, const std::string &argument);
+void	flag_invite_only(bool add_flag, User &author, Channel &channel, const std::string &argument);
+void	flag_key(bool add_flag, User &author, Channel &channel, const std::string &argument);
```

Comment:
If I was a real bully, I would ask you to move the boolean parameter at the end of the list, as it is the less important one according to me and only me... :slightly_smiling_face: